script/Ciel/Day 7/07_01_CIEL07_4.txt

```diff
@@ -103,38 +111,43 @@ Even though I'm not even part of the conversation, Kohaku suddenly gives me the
 [sha:62ed69274c26a2e51fb674e70b5c11c0285860cd]{
 -- Page 405, Offset 15785.
 -- 横目で盗み見た翡翠はどことなく緊張しているように見えた。
-Sneaking a glance at Hisui out of the corner of my eyes reveals her looking somewhat nervous.
+I sneak a glance at Hisui out of the corner of my eye, and notice she's looking somewhat nervous.
 }
 [sha:0916643550a65b84191c3eb76aeb8b2feb4fc155]{
 -- Page 406, Offset 15786.
 -- 「うーん、神経質とは違うような。きめ細かい仕事と神経質なのは別でしょ。俺がポカやってもすぐに片付けてくれるし、自分でも気付かない体調を指摘してくれるし。
-"Hmmm, I feel like fussy isn't quite right. Being meticulous in your work is not the same as being fussy, now is it?
- I mean, whenever I mess something up, she immediately takes care of it for me;
- whenever I'm unaware of my physical condition, she points it out for me;
+"Hmm, I feel like 'fussy' is the wrong word.
+ Being meticulous in your work isn't the same as being fussy, now is it?
+ I mean, whenever I mess something up she immediately takes care of it for me,
+ and when I'm feeling under the weather but haven't picked up on it yet she points it out for me.
 }
 [sha:ad4cc9b9d148aa75bde23d90627b5fdd0e9935b9]{
 -- Page 406, Offset 15787.
 -- あと夜遅くに帰っても誰かさんみたいに怒らないし。そういうのはおとなしい、優しい、と表現すべきだよ」
-And finally, whenever I come home late at night she doesn't get mad at me, unlike a certain somebody.
- That's more appropriately expressed as 'well-behaved' and 'kind', honestly."
+On top of that, she doesn't get mad at me when I come home late in the evening, unlike a certain somebody.
+ I think you'd call that 'well-behaved' and 'kind', if anything."
```

Comment:
I would replace 'well behaved' with gentle. Feels a little arrogant for Shiki to say.